addon/helpers/list.js

```diff
@@ -2,6 +2,16 @@ import { MicroState } from 'ember-microstates';
 
 export default MicroState.extend({
 
+  initialValueFor([array]) {
+    if (array === undefined) {
+      return [];
+    } else if (array && array.length != null && array.forEach) {
```

Comment:
I think `instanceof Array` should suffice, we can always add more scenarios in the future.